src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.cpp

```diff
@@ -0,0 +1,519 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "HostFxrResolver.h"
```

Comment:
I still see hostfxr_utility.cpp/.h in the folder (removed from sln but not deleted). I think it's just a rename so I'm going to ignore all other changes in this file.